src/Serilog.Sinks.Debug/Sinks/Debug/DebugSink.cs

```diff
@@ -34,7 +34,7 @@ public void Emit(LogEvent logEvent)
             using (var buffer = new StringWriter())
             {
                 _formatter.Format(logEvent, buffer);
-                System.Diagnostics.Debug.WriteLine(buffer.ToString());
+                System.Diagnostics.Debug.WriteLine(buffer.ToString().Trim());
```

Comment:
Instead of trimming off the newline, we should use `Debug.Write()` - the use of `WriteLine()` here is a mistake.